echopype/convert/set_groups_azfp.py

```diff
@@ -8,13 +8,15 @@
 from ..utils.coding import set_encodings
 from .set_groups_base import SetGroupsBase
 
-_varattrs = sonarnetcdf_1.conv.yaml_dict["variable_and_varattributes"]
-DEFAULT_BEAM_COORD_ATTRS = _varattrs["beam_coord_default"]
-
 
 class SetGroupsAZFP(SetGroupsBase):
     """Class for saving groups to netcdf or zarr from AZFP data files."""
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self._varattrs = sonarnetcdf_1.conv.yaml_dict["variable_and_varattributes"]
```

Comment:
I like how this works @emiliom. Though one question I have is how come this is not inserted within `SetGroupsBase`? It seems that `self._varattrs` is used in each SetGroups.

Reply:
I didn't think of that! It sounds like a great suggestion. I'll look into it soon.